calcom/cal.com
# Grab specific credentials when cancelling bookings

## What does this PR do?



This PR refactors how when cancelling bookings calls are made to calendar and video integrations.

This PR relies on changes made in #3315 #3375

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Connect two calendars of the same type
- Create a booking and immediately cancel
	- In the logs you shouldn't see any errors
- Create an other booking
- In the booking reference for the calendar delete the credential id
- Now cancel the booking	
- In the logs you should see two calendar requests, one successful and one 404

## Checklist




- I haven't checked if new and existing unit tests pass locally with my changes
